modules/swagger-codegen/src/main/resources/lua/partial_header.mustache

```diff
@@ -1,13 +1,13 @@
-/* 
+--[[ 
  {{#appName}}
- * {{{appName}}}
- *
+  {{{appName}}}
+ 
  {{/appName}}
  {{#appDescription}}
- * {{{appDescription}}}
- *
+  {{{appDescription}}}
+ 
  {{/appDescription}}
- * {{#version}}OpenAPI spec version: {{{version}}}{{/version}}
- * {{#infoEmail}}Contact: {{{infoEmail}}}{{/infoEmail}}
- * Generated by: https://github.com/swagger-api/swagger-codegen.git
- */
+  {{#version}}OpenAPI spec version: {{{version}}}{{/version}}
+  {{#infoEmail}}Contact: {{{infoEmail}}}{{/infoEmail}}
+  Generated by: https://github.com/swagger-api/swagger-codegen.git
+--]]
```

Comment:
We need to escape that to avoid code injection as discovered by a security researcher before.